person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    self.id = Random.ran(1..1000)
+    self.name = name
+    self.age = age
+    self.parent_permission = parent_permission
```

Comment:
- [x] Kindly make use of the `instance variable` here instead of using self. Also, there is a typo error where you're generating the random numbers for `id`, you have `Random.ran(1..1000)` instead of `Random.rand(1..1000). Hints;

```suggestion
    @id = Random.rand(1..1000)
    @name = name
    @age = age
    @parent_permission = parent_permission
```

Reply:
Done